README.md

```diff
@@ -102,503 +102,77 @@ identifiable. If you are prototyping, you can use `java.util.UUID` to generate a
 In most cases, `clientId` can be set to `userId`, the user’s application-specific identifier, provided that `userId`
 is unique within the context of your application.
```

Comment:
Can we add this statement to the official web documentation, as the use of clientId is language-agnostic?

`
In most cases, `clientId` can be set to `userId`, the user’s application-specific identifier, provided that `userId`
is unique within the context of your application.
`

Link to internal convo -> https://ably-real-time.slack.com/archives/CURL4U2FP/p1732882511584999

Reply:
Will look into this outside of this PR

Reply:
Please do 👍